README.md

```diff
@@ -1,2 +1,15 @@
 # blog
 Blog posts from PH for PH and the rest of the world :)
+
+Formating of blog posts:
+
+When you first make a blog post make sure to have a JSON header stating your name, title of your blog post, date, tags, and publication. 
+The JSON should also be contained in brackets. An example of this is down below. 
+
+{ "author": "Riyan Imam", "title": "HackATL: Busy Making Businesses!", "date": "2016-11-08T23:35:24Z", "tags": ["all"], "published": true
```

Comment:
Can you style the JSON better per conventions? So each attribute and value in it's own line